Makefile

```diff
@@ -261,10 +262,19 @@ deploy_nodes_with_install:
 	$(bash scripts/utils.sh local_setup_before_deployment $(PLATFORM) $(NAMESPACE) $(OC_FLAG))
 	skipper make $(SKIPPER_PARAMS) _deploy_nodes NAMESPACE_INDEX=$(shell bash scripts/utils.sh get_namespace_index $(NAMESPACE) $(OC_FLAG)) NAMESPACE=$(NAMESPACE) ADDITIONAL_PARAMS="'-in ${ADDITIONAL_PARAMS}'" $(SKIPPER_PARAMS) DAY1_PARAMS=--day1-cluster
 
+
+deploy_static_ips_nodes_with_install:
+	$(bash scripts/utils.sh local_setup_before_deployment $(PLATFORM) $(NAMESPACE) $(OC_FLAG))
+	skipper make $(SKIPPER_PARAMS) _deploy_nodes NAMESPACE_INDEX=$(shell bash scripts/utils.sh get_namespace_index $(NAMESPACE) $(OC_FLAG)) NAMESPACE=$(NAMESPACE) ADDITIONAL_PARAMS="'-in -sic $(STATIC_IPS_CONFIG)'" $(SKIPPER_PARAMS) DAY1_PARAMS=--day1-cluster
+
 deploy_nodes:
 	$(bash scripts/utils.sh local_setup_before_deployment $(PLATFORM) $(NAMESPACE) $(OC_FLAG))
 	skipper make $(SKIPPER_PARAMS) _deploy_nodes NAMESPACE_INDEX=$(shell bash scripts/utils.sh get_namespace_index $(NAMESPACE) $(OC_FLAG)) NAMESPACE=$(NAMESPACE) DAY1_PARAMS=--day1-cluster
 
+deploy_static_ips_nodes:
+	$(bash scripts/utils.sh local_setup_before_deployment $(PLATFORM) $(NAMESPACE) $(OC_FLAG))
+	skipper make $(SKIPPER_PARAMS) _deploy_nodes NAMESPACE_INDEX=$(shell bash scripts/utils.sh get_namespace_index $(NAMESPACE) $(OC_FLAG)) NAMESPACE=$(NAMESPACE) DAY1_PARAMS=--day1-cluster ADDITIONAL_PARAMS="'-sic $(STATIC_IPS_CONFIG)'"
```

Comment:
can't we just set ADDITIONAL_PARAMS="'-sic $(STATIC_IPS_CONFIG)'" and run deploy_nodes ?

Reply:
i wanted a different target, so that user does not needs to know what additional parameters to set, more convenient

Reply:
i mean if this target can call deploy_nodes and just provide it's own ADDITIONAL_PARAMS, without duplicating same code

Reply:
this is what it does, look at all the other targets.  ADDITIONAL_PARAMS, Namespace and OC_FLAG

Reply:
fixed